rules/alerting.go

```diff
@@ -83,7 +83,9 @@ type Alert struct {
 	Value float64
 	// The interval during which the condition of this alert held true.
 	// ResolvedAt will be 0 to indicate a still active alert.
-	ActiveAt, ResolvedAt time.Time
+	ActiveAt   time.Time
+	FiringAt   time.Time
```

Comment:
Maybe `LastFiredAt`? At first glance I wasn't sure what this was going to be.

Reply:
Mh, it's not the last fired at timestamp though but the timestamp at which it first started firing.

Reply:
Right, then `FirstFiredAt`?